docs/man/git-lfs-config.5.ronn

```diff
@@ -3,12 +3,16 @@ git-lfs-config(5) -- Configuration options for git-lfs
 
 ## CONFIGURATION FILES
 
-git-lfs reads its configuration from a file called `.lfsconfig` at the root of
-the repository. The `.lfsconfig` file uses the same format as `.gitconfig`.
+git-lfs reads its configuration from any file supported by `git config -l`,
+including all per-repository, per-user, and per-system Git configuration files.
 
-Additionally, all settings can be overridden by values returned by `git config -l`.
-This allows you to override settings like `lfs.url` in your local environment
-without having to modify the `.lfsconfig` file.
+Additionally, a small number of settings can be specified in a file called
```

Comment:
I think with this change it's no longer clear what `.lfsconfig` should be used for, as we're now suggesting users primarily use normal `.gitconfig` files and settings in those files will explicitly override the ones in `.lfsconfig`.

Maybe we could add a note about what the possible use cases for `.lfsconfig` are?

Reply:
Sure, that sounds good.

Reply:
Okay, I've updated this text to provide a description of why users would want to use the `.lfsconfig` file.  Let me know if there's another use case I should mention or something I've left out.